zebra-consensus/src/checkpoint.rs

```diff
@@ -393,12 +393,12 @@ where
     /// Increase the current checkpoint height to `verified_height`,
     fn update_progress(&mut self, verified_height: block::Height) {
         if let Some(max_height) = self.queued.keys().next_back() {
-            metrics::gauge!("checkpoint.queued.max.height", max_height.0 as i64);
+            metrics::gauge!("checkpoint.queued.max.height", max_height.0 as f64);
         } else {
             // use -1 as a sentinel value for "None", because 0 is a valid height
-            metrics::gauge!("checkpoint.queued.max.height", -1);
+            metrics::gauge!("checkpoint.queued.max.height", -1.0);
```

Comment:
Since this is just a sentinel value, what happens if we use `NaN` here?
```suggestion
            metrics::gauge!("checkpoint.queued.max.height", f64::NAN);
```